tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -336,3 +351,61 @@ def test_positive_custom_user_view_lce(self):
                 session.nav.go_to_users()
             # assert that the user can view the lvce created by admin user
             self.assertIsNotNone(self.lifecycleenvironment.search(env_name))
+
+    @run_only_on('sat')
+    @tier3
+    def test_positive_search_lce_content_view_packages(self):
+        """Search Lifecycle Environment content view packages
+
+        :id: bceb97ac-cd20-4a45-a250-f26c194f651a
+
+        :steps:
+            1. Create a product with a repository synchronized
+                - The repository must contain at least two package names P1 and
+                  P2
+                - P1 has only one package version
+                - P2 has two package versions
+            2. Create a content view with the repository and publish it
+            3. Go to Lifecycle Environment > Library > Packages
+            4. Select the content view
+            5. Search by exact package names with versions
+            6. Search by package names
+
+        :expectedresults: only the searched packages where found
+
+        :BZ: 1432155
+
+        :CaseLevel: System
+        """
+        packages = [
+            {'name': FAKE_0_CUSTOM_PACKAGE_NAME,
+             'versions': [FAKE_0_CUSTOM_PACKAGE]},
+            {'name': FAKE_1_CUSTOM_PACKAGE_NAME,
+             'versions': [FAKE_1_CUSTOM_PACKAGE, FAKE_2_CUSTOM_PACKAGE]},
+        ]
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        repository = entities.Repository(
+            product=product, url=FAKE_0_YUM_REPO).create()
+        repository.sync()
+        content_view = entities.ContentView(
+            organization=org, repository=[repository]).create()
+        content_view.publish()
+        with Session(self) as session:
+            set_context(session, org=org.name)
+            for package in packages:
+                for package_version in package['versions']:
+                    names_found = self.lifecycleenvironment.get_yum_packages(
+                        ENVIRONMENT,
+                        package_version,
```

Comment:
method receives package name, not package version. In case it do, we should have proper parameters name and docstring. Please, expand docstring with each parameter explanation

Reply:
package_version is the full package name with version and arch
for example: 
package name = bear
package version = bear-4.1-1.noarch

for package name = walrus
it has two package versions walrus-0.71-1.noarch and walrus-5.21-1.noarch